app/web/features/auth/useAuthStore.ts

```diff
@@ -161,14 +162,14 @@ export default function useAuthStore() {
               action: "updateJailStatus",
             },
           });
-          setError(isGrpcError(e) ? e.message : t("fatal_error_message"));
+          setError(isGrpcError(e) ? e.message : fatalErrorMessage.current);
         }
         setLoading(false);
       },
     }),
-    //note: there should be no dependenices on the state, or
+    //note: there should be no dependenices on the state or t, or
```

Comment:
That might explain why I run into funky issues where all of the tests failed because of this! Kinda disappointing frameworks don't think about the importance of keeping their function stable if possible, especially as this is implemented with hooks so you'd thought they expect people to use this in an `useEffect` in some places...